mongodbatlasv2/api_access_tracking.go

```diff
@@ -71,6 +71,16 @@ type AccessTrackingApiListAccessLogsByClusterNameRequest struct {
 	start *time.Time
 }
 
+type AccessTrackingApiListAccessLogsByClusterNameQueryParams struct {
+		GroupId string
+		ClusterName string
```

Comment:
group id and cluster name are path param, is this the expected usage?

Reply:
Yes. We need all arguments (file path params and query params in one place. 
I did failed naming the structure correctly thought. 
Will use `Opts` to make it more backwards compatible with v1